src/mca/schizo/ompi/schizo_ompi.c

```diff
@@ -1256,7 +1256,7 @@ static int parse_env(char **srcenv, char ***dstenv,
             p3 = strchr(opt->values[i], '=');
             *p3 = '\0';
             ++p3;
-            p1 = opt->values[0];
+            p1 = opt->values[1];
```

Comment:
Should be `opt->values[i]`

Reply:
Yes that seems to work a lot better. thanks! updated the pr.